src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -157,6 +157,48 @@ private String convertTopicToIndexName(String topic) {
     return index;
   }
 
+  /**
+   * Returns the converted index name from a given topic name in the form {type}-{dataset}-{topic}.
+   * For the <code>topic</code>, Elasticsearch accepts:
+   * <ul>
+   *   <li>all lowercase</li>
+   *   <li>no longer than 100 bytes</li>
```

Comment:
the whole data stream name is less than 100 bytes or just the namespace/topic portion?

Reply:
Just the namespace/topic portion. I only mentioned the topic because it's the only parameter and the other restrictions are validated in the configs. Should I update the method header to talk about the other fields too?

Reply:
nope its okay, I was just clarifying for myself